frontend/src/components/core/Text.tsx

```diff
@@ -15,3 +15,7 @@ export const LargeText = styled.h3`
 export const LandingHeaderText = styled.h1`
   font-size: ${({ theme }) => theme.fontSize.xxLarge};
 `;
+
+export const ProblemHeaderText = styled.h3`
+  font-size: ${({ theme }) => theme.fontSize.Medium};
```

Comment:
Medium is a font size, but it's the default value so technically line 20 is not necessary. I kept it there just to have a `ProblemHeaderText` because we'll probably need more styling on it later.

Reply:
Where do you see Medium is a font size? The below image shows the font sizes I can see -- where is Medium? If it is equal to `default`, why not just use `default` like we have elsewhere?
<img width="213" alt="Medium Font Size Not Found" src="https://user-images.githubusercontent.com/7987279/93163030-f572f580-f6ca-11ea-845d-f17e50efb69c.png">

Reply:
Good point, I'll fix that.

Reply:
Has this been fixed yet?